protocol/app/app.go

```diff
@@ -66,6 +66,9 @@ import (
 	distr "github.com/cosmos/cosmos-sdk/x/distribution"
 	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
 	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/cosmos/cosmos-sdk/x/evidence"
+	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
+	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
```

Comment:
Given that the `x/evidence` defines a few msg txs ([code](https://github.com/cosmos/cosmos-sdk/blob/79a64d07d0bf2964b3c38f11241c90a84b2f1262/proto/cosmos/evidence/v1beta1/tx.proto)), I suspect you will need to update a few files and tests in `app/msgs` [here](https://github.com/dydxprotocol/v4-chain/tree/6d30844950ba73e1d1003e11f320e1ee3529ca70/protocol/app/msgs).

Can you run the unit tests there and see if you need to update anything there?

Reply:
Fixed tests